### PR TITLE
Fix the error FormElement Iframe Has Submit Button

### PR DESCRIPTION
**Pull request by sureshkumarbandaru**

### CODE REVIEW

**Comment by seangates** [comment]
Shouldn't the file name rename be `1.1.33` (not `1.1.3`)?

File: rulesets/custom.ruleset.1.1.3.js [hunk 1]
```diff
@@ -463,10 +463,12 @@ function formElementIFrameHasSubmitButton(formElement) {
...
-		var innerDoc = iframeElement.contentDocument || iframeElement.contentWindow.document;
-		if(formElementFindNonHiddenInputElement(innerDoc, innerDoc, ".//*[(local-name()='input' and @type='submit') or (local-name()='input' and @type='image') or (local-name()='button' and @type='submit')]") != null) {
-			return true;
-		}
+		try{
+		    var innerDoc = iframeElement.contentDocument || iframeElement.contentWindow.document;
+		    if(formElementFindNonHiddenInputElement(innerDoc, innerDoc, ".//*[(local-name()='input' and @type='submit') or (local-name()='input' and @type='image') or (local-name()='button' and @type='submit')]") != null) {
+		    return true;
+		    }
+		}catch(err){}
```
Comment (seangates) [inline rulesets/custom.ruleset.1.1.3.js]:
```suggestion
		} catch(err) {}
```

File: rulesets/custom.ruleset.1.1.33.js [hunk 2]
```diff
@@ -463,10 +463,12 @@ function formElementIFrameHasSubmitButton(formElement) {
...
-		var innerDoc = iframeElement.contentDocument || iframeElement.contentWindow.document;
-		if(formElementFindNonHiddenInputElement(innerDoc, innerDoc, ".//*[(local-name()='input' and @type='submit') or (local-name()='input' and @type='image') or (local-name()='button' and @type='submit')]") != null) {
-			return true;
-		}
+		try{
+		    var innerDoc = iframeElement.contentDocument || iframeElement.contentWindow.document;
+		    if(formElementFindNonHiddenInputElement(innerDoc, innerDoc, ".//*[(local-name()='input' and @type='submit') or (local-name()='input' and @type='image') or (local-name()='button' and @type='submit')]") != null) {
+		    return true;
+		    }
+		}catch(err){}
```
Comment (seangates) [inline rulesets/custom.ruleset.1.1.33.js]:
```suggestion
		} catch(err) {}
```

**Comment by scottizu** [comment]
Please run tests as described here: https://github.com/eBay/accessibility-ruleset-runner/blob/master/rulesets/tests/README.md

All contributors are required to run tests prior to commit.

**Comment by scottizu** [comment]
https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/nodejs/package.json#L15 shows 1.0.0 by default, but it is not used.

Update https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/nodejs/custom.ruleset.runner.js#L40 to use 1.1.33

We use relative paths in custom.ruleset.runner.js and also do not require the package.json dependency.

**Comment by scottizu** [comment]
For Test NG Example, update this line: https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/java/src/test/java/arr/RulesetDownloader.java#L26

**Comment by scottizu** [comment]
Chrome Extension Example: https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/chromeextension/src/content/rulesetRunner.js#L19

**Comment by scottizu** [comment]
Python Example: 

https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/python/all.ruleset.runner.py#L92

https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/python/all.ruleset.runner.py#L99

https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/python/all.ruleset.runner.py#L109

https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/python/all.ruleset.runner.py#L113

https://github.com/eBay/accessibility-ruleset-runner/blob/master/examples/python/custom.ruleset.runner.py#L39

**Comment by scottizu** [comment]
It might be good to auto detect the latest file for these examples but for now, every new version of the custom ruleset or axe ruleset will require such changes.

Also, potentially publishing a new NPM Package.

Potentially, updating output documentation for examples/tests, Chrome Extension as well.

**Comment by scottizu** [comment]
https://github.com/eBay/accessibility-ruleset-runner/issues/19

**Pull request merged**